broker/src/main/java/io/zeebe/broker/system/partitions/ZeebePartition.java

```diff
@@ -485,6 +486,22 @@ private void installProcessingPartition(final CompletableActorFuture<Void> insta
             });
   }
 
+  private void installRocksDBMetricExporter(final ZeebeRocksDBMetricExporter metricExporter) {
+    final var metricsTimer =
+        actor.runAtFixedRate(
+            Duration.ofSeconds(5),
+            () -> {
+              if (zeebeDb != null) {
```

Comment:
Who closes `zeebeDb`? `ZeebePartition`? `StateSnapshotController`? Just wondering if this null check is enough.

Also, a little out of scope but since we're here, should we rather reschedule the next metrics export here instead of using `runAtFixedRate`? We've ran into issues before where you may build up a lot of tasks with `runAtFixedRate`, no? Though I guess here our actor is usually not very busy, so :man_shrugging: